system/vmware-monitoring/templates/collector-configmap.yaml

```diff
@@ -432,6 +434,8 @@ data:
         key: "net|bytesRx_average"
       - metric_suffix: "network_data_transmitted_kilobytes_per_second"
         key: "net|bytesTx_average"
+      - metric_suffix: "network_usage_rate_kilobytes_per_second"
+        key: "net|usage_average"
```

Comment:
That's a duplicate entry.
The key is already present.
You can use the available metric.
```suggestion
```